standalone/runtime/src/frontier_evm.rs

```diff
@@ -110,15 +110,13 @@ impl pallet_evm_precompile_proxy::EvmProxyCallFilter for ProxyType {
 
 /// Current approximation of the gas/s consumption considering
 /// EVM execution over compiled WASM (on 4.4Ghz CPU).
-/// Given the 500ms Weight, from which 75% only are used for transactions,
-/// the total EVM execution gas limit is: GAS_PER_SECOND * 0.500 * 0.75 ~= 15_000_000.
-pub const GAS_PER_SECOND: u64 = 40_000_000;
+pub const GAS_PER_SECOND: u64 = 400_000_000;
 
 /// Approximate ratio of the amount of Weight per Gas.
 /// u64 works for approximations because Weight is a very small unit compared to gas.
 pub const WEIGHT_PER_GAS: u64 = WEIGHT_REF_TIME_PER_SECOND / GAS_PER_SECOND;
 
-const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
+const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(85);
```

Comment:
is this safe? is there any other network changing this? could be risky since upgrades/hooks are operational and this will reduce their block space to 15%
https://github.com/paritytech/substrate/pull/5954

Reply:
Okay, actually let me lower this back to 75% and try again.